### ISSUE TITLE
RPC[Error] code_name = InvalidParams, message = "Document for URI could not be found: ..."

### ISSUE THREAD

**Issue by fredrikaverpil** [MagicDuck/grug-far.nvim#93]
Hey @MagicDuck and thanks for this fantastic plugin! 👏 😄
I've always struggled a bit with [nvim-spectre](https://github.com/nvim-pack/nvim-spectre) in terms of how you have to mix filters and flags, and your UI and keymaps just makes everything so much more intuitive to me. Instant replacement here! ⭐ 

I am however getting an RPC error for each character I type into the search field. This is me typing in `hello` into the search field:

```
08:35:31 msg_show RPC[Error] code_name = InvalidParams, message = "Document for URI could not be found: file:///Users/fredrik/code/dotfiles/Grug%20FAR%20-%201%3A%20he, URIs of the known document are: file:///Users/fredrik/code/dotfiles/Grug%20FAR%20-%201"
08:35:33 msg_show RPC[Error] code_name = InvalidParams, message = "Document for URI could not be found: file:///Users/fredrik/code/dotfiles/Grug%20FAR%20-%201%3A%20hel, URIs of the known document are: file:///Users/fredrik/code/dotfiles/Grug%20FAR%20-%201"
08:35:34 msg_show RPC[Error] code_name = InvalidParams, message = "Document for URI could not be found: file:///Users/fredrik/code/dotfiles/Grug%20FAR%20-%201%3A%20hell, URIs of the known document are: file:///Users/fredrik/code/dotfiles/Grug%20FAR%20-%201"
08:35:36 msg_show RPC[Error] code_name = InvalidParams, message = "Document for URI could not be found: file:///Users/fredrik/code/dotfiles/Grug%20FAR%20-%201%3A%20hello, URIs of the known document are: file:///Users/fredrik/code/dotfiles/Grug%20FAR%20-%201"
```

My current working directory is `/Users/fredrik/code/dotfiles` and it feels like grug-far is expecting (or looking for) a file in this location, matching the pattern `Grug FAR - 1: <search term>`.

This doesn't appear to affect functionality other than being annoying (as I get popups about this).

System:
- macOS
- Wezterm
- Neovim 0.10
- Grug-FAR config [here](https://github.com/fredrikaverpil/dotfiles/commit/ad92279671ebf247b93af1bd2ee1220eb180f189)
- Zsh
- rg version
  ```
  ❯ rg --version
  ripgrep 14.1.0

  features:-simd-accel,+pcre2
  simd(compile):+NEON
  simd(runtime):+NEON

  PCRE2 10.42 is available (JIT is available)
  ```

**Comment by MagicDuck** [comment]
HI Frederik, thank you for the kind words and detailed info! 😄 

That error message makes me suspicious that a language server is trying to attach to the buffer, but there is no backing file for it naturally.
If you run `:LspInfo` while in the grug-far.nvim buffer, do you see any language server attached?
<img width="2054" alt="image" src="https://github.com/MagicDuck/grug-far.nvim/assets/95201/391f4c2a-46b8-43b6-96a6-dc385a1b1fbd">

**Comment by MagicDuck** [comment]
could also be copilot / other AI related if you use that...

**Comment by fredrikaverpil** [comment]
Ah yes, that was it - thank you!
I have `copilot.lua` activated for public git repos, and I don't see this error in the non-public ones. 😄 

I just noticed this didn't work well in Lua as it doesn't like the dash:

```lua
filetypes = {
  grug-far = false,
  grug-far-history = false,
}
```

I worked around it with `opts.filetypes["grug-far"] = false` but had to place this outside my opts config. With this, no more rpc errors! 👍 
Thanks again so much for the support and your work on the plugin! ⭐
